### PR TITLE
29 integrate calendar events

### LINKED ISSUE #29: Integrate Calendar /Events

### LINKED ISSUE #45: Account page - change "session participation" from t/f to list of participated sessions.
image

### LINKED ISSUE #46: cannot register for event unless user accepts CoC
disable "register" button "agree to CoC in your profile page" underneath

### LINKED ISSUE #47: Create event Registration process
- button on "register" event that will add it to many-to-many between user and event obj

### PR DESCRIPTION
**Pull request by RachellCalhoun**
closes #29 
closes #47 
closes #46 
closes #45 
I saw the calendar /event cards after I created the event detail etc. But I figured we can migrate later into the existing templates.

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
I plugged in fullcalendar.io on the front end to display events, click on the even and it will take you to the detail page. 
Screenshot from 2023-01-04 14-58-25

**Comment by RachellCalhoun** [comment]
@dawnwages  check this out: 
- added color coding to the calendar page (pending is hidden, canceled is grey, rescheduled is orange, and scheduled is default blue) 
- order by start time 
- improved django admin for adding users/speakers/categories/organizers
- added RSVP button:
    -   register for non authenticated users
    -   disabled RSVP button for users that haven't accepted CoC
    -  Cancel RSVP button for folks who have already RSVPed 
- added speaker photo, bio, and contacts below the details page 
Check out the video: 

https://www.loom.com/share/620a6ff0cb124be682b35bfdaa358d43

File: indymeet/urls.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
     path("search/", search_views.search, name="search"),
     # path("home/", home_views.test_page, name="test"),
     path("accounts/", include("accounts.urls")),
+    path("", include("home.urls")),
 ]
```
Comment (RachellCalhoun) [inline indymeet/urls.py:19]: @dawnwages  I know you used "" for the pupet URLS but I think we should reserve that space for our home.urls and , if I'm not mistaken ...pupet is a blog? So maybe we could have something more descriptive there like "blog/" ..?

Comment (dawnwages) [reply indymeet/urls.py:19]: sorry, I'm just realizing why you said this. #41 has already been merged. You can add it here in this PR. Thanks, you're totally right.

Comment (dawnwages) [reply indymeet/urls.py:19]: and I see addressing this also resolves your conflict. I'm down for this changed and can be merged after this. 👍🏾

**Review by dawnwages — APPROVED** [review]
Perfect!

**Pull request merged**